superset/connectors/sqla/models.py

```diff
@@ -931,27 +931,30 @@ def _get_sqla_row_level_filters(
 
     def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-many-branches,too-many-statements
         self,
-        metrics: Optional[List[Metric]] = None,
-        granularity: Optional[str] = None,
-        from_dttm: Optional[datetime] = None,
-        to_dttm: Optional[datetime] = None,
+        apply_fetch_values_predicate: bool = False,
         columns: Optional[List[str]] = None,
-        groupby: Optional[List[str]] = None,
+        extras: Optional[Dict[str, Any]] = None,
         filter: Optional[  # pylint: disable=redefined-builtin
             List[QueryObjectFilterClause]
         ] = None,
-        is_timeseries: bool = True,
-        timeseries_limit: int = 15,
-        timeseries_limit_metric: Optional[Metric] = None,
-        row_limit: Optional[int] = None,
-        row_offset: Optional[int] = None,
+        from_dttm: Optional[datetime] = None,
+        granularity: Optional[str] = None,
+        groupby: Optional[List[str]] = None,
         inner_from_dttm: Optional[datetime] = None,
         inner_to_dttm: Optional[datetime] = None,
+        is_rowcount: bool = False,
+        is_timeseries: bool = True,
+        metrics: Optional[List[Metric]] = None,
         orderby: Optional[List[OrderBy]] = None,
-        extras: Optional[Dict[str, Any]] = None,
         order_desc: bool = True,
-        is_rowcount: bool = False,
-        apply_fetch_values_predicate: bool = False,
+        to_dttm: Optional[datetime] = None,
+        series_columns: Optional[List[str]] = None,
+        series_limit: Optional[int] = None,
+        series_limit_metric: Optional[Metric] = None,
+        row_limit: Optional[int] = None,
+        row_offset: Optional[int] = None,
+        timeseries_limit: Optional[int] = None,
```

Comment:
`timeseries_limit` previously defaulted to `15`. However, this was always defaulted to `0` in both `viz.py` and `QueryObject`, so we never actually used this magic number.